.editorconfig

```diff
@@ -156,7 +156,7 @@ dotnet_diagnostic.IDE0005.severity = warning
 # RS0041: Public members should not use oblivious types
 dotnet_diagnostic.RS0041.severity = suggestion
 
-[obj/**.cs]
+[**/obj/**.cs]
```

Comment:
@rajkumar-rangaraj This cleans up those proto warnings you had to suppress in project files.